### DISCUSSION TITLE
Open image editor after upload and require a crop ratio

### DISCUSSION THREAD

**Discussion by sweebee** [filamentphp/filament#9127] [Ideas]
It would be nice to allow the imageEditor to open when the upload is complete. It would also be nice that the user has to choose a crop factor before submitting the image.

In my case the user can upload an image and this image has to be 16:9 or 21:9. So the user can upload an image with this ratio or he uploads a different ratio and the user has to choose to crop it to 16:9 or 21:9.

**Comment by mstfkhazaal** [comment]
1+

**Comment by abetterlogic** [comment]
+1

**Comment by kologlu61** [comment]
+1

**Comment by mhortulanus** [comment]
+1

**Comment by Baspa** [comment]
+1

**Comment by danharrin** [comment]
https://filamentphp.com/docs/4.x/forms/file-upload#enforcing-a-specific-aspect-ratio
